src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * Add `CamelCaseToSnakeCaseNameConverter::REQUIRE_SNAKE_CASE_PROPERTIES` context option
  * Deprecate `AbstractNormalizerContextBuilder::withDefaultContructorArguments(?array $defaultContructorArguments)`, use `withDefaultConstructorArguments(?array $defaultConstructorArguments)` instead (note the missing `s` character in Contructor word in deprecated method)
  * Add `XmlEncoder::CDATA_WRAPPING_PATTERN` context option
+ * Add headers label support for `CsvEncoder` using `csv_headers`
```

Comment:
Please move this to a new 7.2 section, thanks

Reply:
I was hoping I still have a chance to see it merged in 7.1 branch since the final version is not released yet.

Reply:
Done